brianlong/www.validators.app
# 187087883 - update jito pool with correct logo

#### What's this PR do?
- Replace with correct logo for Jito.

#### How should this be manually tested?
- verify /stake-pools and main page if the new logo is displayed
- at /validators check for Jito badge. It should be the same as before.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187087883)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
